docs/index.md

```diff
@@ -6,180 +6,36 @@ nav_order: 0
-
-#### Cross-collection consistency guarantees
-
-Data engineers typically need to implement custom logic in scripts to guarantee
-two or more data assets are updated synchronously.  This logic often
-requires extensive rewrites or periods during which data is unavailable.
-The lakeFS merge operation from one branch into another removes the need to
-implement this logic yourself.
-
-Instead, make updates to the desired data assets on a branch, and then utilize a lakeFS merge to atomically expose the data to downstream consumers.
-
-To learn more about atomic cross-collection updates, [this video](https://www.youtube.com/watch?v=9OsjUvk5UJU) describes the concept in more detail.
-
-#### Troubleshooting production problems
-
-Data engineers are often asked to validate the data. A user might report inconsistencies, question the accuracy, or simply report it to be incorrect. Since the data continuously changes, it is challenging to understand its state at the time of the error.
-
-The best way to investigate, therefore, is to have a snapshot of the data as close as possible to the time of the error.
-Once implementing a regular commit cadence in lakeFS, each commit represents an accessible historical snapshot of the data. When needed, a branch may be created from a commit ID to debug an issue in isolation.
-
-To learn more on how to access a specific historical commit in a repository, see our seminal post on [data reproducibility](https://lakefs.io/solving-data-reproducibility/).
-
-#### Establishing data quality guarantees
-
-The best way to deal with mistakes is to avoid them. A data source that is ingested into the lake introducing low-quality data should be blocked before exposure if possible.
-
-With lakeFS, you can achieve this by tying data quality tests to commit and merge operations via lakeFS [hooks](./setup/hooks.md).
-
-
-### Additional things you should know about lakeFS: 
-
-* It is format agnostic
-* Your data stays in place
-* It minimizes data duplication via a copy-on-write mechanism
-* It maintains high performance over data lakes of any size
-* It includes configurable garbage collection capabilities
-* It is highly available and production ready
-
-<img src="{{ site.baseurl }}/assets/img/lakeFS_integration.png" alt="lakeFS integration into data lake" width="60%" height="60%" />
-
-
-### Downloads
-
-#### Binary Releases
-
-Binary packages are available for Linux/macOS/Windows on [GitHub Releases](https://github.com/treeverse/lakeFS/releases){: target="_blank" }
-
-#### Docker Images
-
-Official Docker images are available at [https://hub.docker.com/r/treeverse/lakefs](https://hub.docker.com/r/treeverse/lakefs){: target="_blank" }
-
+> df = spark.read.parquet("s3a://my-repo/main-branch/collections/foo/")
 
-### Next steps
```

Comment:
Why remove the next steps?

Reply:
did I remove it completely or move it into another doc?